packages/jest-config/src/loadFromFile.js

```diff
@@ -26,8 +26,7 @@ function loadFromFile(filePath: Path, argv: Object) {
       } catch (e) {
         const error = jsonlint.errors(data.toString());
         throw new Error(
-          `Jest: Failed to parse config file ${filePath}\n` +
-          `  ${error}`,
+          `Jest: Failed to parse config file ${filePath}\n  ${error}`,
```

Comment:
You got me! No, it didn't. It did ` + `  ${error}` and then eslint kicked in and was upset about it.